pdsfile/pds4file/rules/cassini_iss.py

```
@@ -286,6 +286,160 @@
+        # bundle xml and other non browse_raw & data_row collections
+        *ARCHIVE_PATHS_DICT['cassini_iss_cruise']['other_col'],
+        # browse_raw
+        *ARCHIVE_PATHS_DICT['cassini_iss_cruise']['browse_raw'],
+        # data_row
+        *ARCHIVE_PATHS_DICT['cassini_iss_cruise']['data_raw'],
+    ]),
+    # input path is a bundle collection path
+    # bundle xml, context, document, and xml_schema
+    (r'.*(bundles|metadata|previews|diagrams)/(cassini_iss/cassini_iss_cruise)/(context|document|xml_schema|bundle.xml)', 0,
+        ARCHIVE_PATHS_DICT['cassini_iss_cruise']['other_col']
+    ),
```

Flake8 (7.2.0)

[error] closing bracket does not match visual indentation

(E124)